Fossil segment doesn't count properly files included in a "merge --integrate" operation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

The `fossil` segment reports incorrectly the number of files added or modified after a `merge --integrate` operation is performed. (This kind of merge differs from a normal merge in that it closes the merged branch to avoid commiting into it.)

The files modified or added in this kind of merge get reported differently in `fossil status` (`ADDED_BY_INTEGRATE` and `UPDATED_BY_INTEGRATE`) and so they don't get counted as additions and changes respectively.

In the reported case, the Fossil segment only shows "trunk +1 -1". It should show "trunk +2 -1 ~1" given the reported

```
ADDED      cinco
ADDED_BY_INTEGRATE cuatro
DELETED    tres
UPDATED_BY_INTEGRATE uno
```

### Theme

Personalized theme:

```json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "version": 2,
  "final_space": true,
  "console_title_template": "{{ .Shell }} in {{ .Folder }}",
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "type": "session",
          "style": "diamond",
          "foreground": "p:white",
          "background": "magenta",
          "background_templates": [
            "{{if .Root}}green{{end}}"
          ],
          "leading_diamond": "\ue0b6",
          "trailing_diamond": "\ue0b0",
          "template": " {{ if .SSHSession }}\ueba9 {{ end }}{{ .UserName }} @ {{ .HostName }} "
        },
        {
          "type": "path",
          "style": "diamond",
          "powerline_symbol": "\ue0b0",
          "trailing_diamond": "\ue0b0",
          "foreground": "p:black",
          "background": "p:orange",
          "template": " \uea83 {{ path .Path .Location }} ",
          "properties": {
            "style": "full"
          }
        },
        {
          "type": "fossil",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "p:black",
          "foreground_templates": [
            "{{ if .Status.Changed }}cyan{{ end }}"
          ],
          "background": "cyan",
          "background_templates": [
            "{{ if .Status.Changed }}p:yellow{{ end }}",
            "{{ if .Status.HasConflicts }}p:red{{ end }}"
          ],
          "properties": {
            "branch_max_length": 25,
            "fetch_status": true,
            "fetch_upstream_icon": true
          }
        },
        {
          "type": "git",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "p:black",
          "foreground_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}p:black{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}p:white{{ end }}",
            "{{ if gt .Ahead 0 }}p:white{{ end }}"
          ],
          "background": "lightGreen",
          "background_templates": [
            "{{ if or (.Working.Changed) (.Staging.Changed) }}p:yellow{{ end }}",
            "{{ if and (gt .Ahead 0) (gt .Behind 0) }}p:red{{ end }}",
            "{{ if gt .Ahead 0 }}#49416D{{ end }}",
            "{{ if gt .Behind 0 }}#7A306C{{ end }}"
          ],
          "template": " {{ if .UpstreamURL }}{{ url .UpstreamIcon .UpstreamURL }} {{ end }}{{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }} ",
          "properties": {
            "branch_max_length": 25,
            "fetch_status": true,
            "fetch_upstream_icon": true
          }
        },
        {
          "type": "root",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "p:white",
          "background": "green",
          "template": " \uf0e7 "
        },
        {
          "type": "shell",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "p:black",
          "background": "p:white",
          "template": " <b>{{ .Name }}</b> "
        },
        {
          "type": "time",
          "style": "diamond",
          "trailing_diamond": "",
          "foreground": "p:black",
          "background": "#9597DD",
          "template": " {{ .CurrentDate | date \"02/01/2006 15:04:05\" }}"
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "right",
      "segments": [
        {
          "type": "node",
          "style": "plain",
          "foreground": "p:green",
          "background": "transparent",
          "template": "\ue718 ",
          "properties": {
            "display_mode": "files",
            "fetch_package_manager": false,
            "home_enabled": false
          }
        },
        {
          "type": "go",
          "style": "plain",
          "foreground": "p:blue",
          "background": "transparent",
          "template": "\ue626 ",
          "properties": {
            "fetch_version": false
          }
        },
        {
          "type": "python",
          "style": "plain",
          "foreground": "p:yellow",
          "background": "transparent",
          "template": "{{ if .Error }}{{ .Error }}{{ else }}\ue235 {{ .Full }} {{ if .Venv }}{{ .Venv }}{{ end }}{{ end }}",
          "properties": {
            "home_enabled": true,
            "display_mode": "context",
            "fetch_version": true,
            "fetch_virtual_env": true
          }
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "foreground": "white",
          "style": "plain",
          "template": "\u276f",
          "type": "text"
        }
      ]
    }
  ],
  "tooltips": [
    {
      "type": "aws",
      "tips": [
        "aws"
      ],
      "style": "diamond",
      "foreground": "p:white",
      "background": "p:orange",
      "leading_diamond": "\ue0b0",
      "trailing_diamond": "\ue0b4",
      "template": " \ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} ",
      "properties": {
        "display_default": true
      }
    },
    {
      "type": "az",
      "tips": [
        "az"
      ],
      "style": "diamond",
      "foreground": "p:white",
      "background": "p:blue",
      "leading_diamond": "\ue0b0",
      "trailing_diamond": "\ue0b4",
      "template": " \uebd8 {{ .Name }} ",
      "properties": {
        "display_default": true
      }
    }
  ],
  "transient_prompt": {
    "foreground": "p:black",
    "background": "transparent",
    "template": "<p:yellow,transparent>\ue0b6</><,p:yellow> {{ .Folder }} </><p:yellow,transparent>\ue0b0</> "
  },
  "secondary_prompt": {
    "foreground": "p:black",
    "background": "transparent",
    "template": "<p:yellow,transparent>\ue0b6</><,p:yellow> > </><p:yellow,transparent>\ue0b0</> "
  },
  "palette": {
    "black": "#262B44",
    "blue": "#4B95E9",
    "green": "#59C9A5",
    "orange": "#FF5501",
    "red": "#D81E5B",
    "violet": "#5B3C9A",
    "white": "#E0DEF4",
    "yellow": "#F3AE35"
  }
}
```

### What OS are you seeing the problem on?

_No response_

### Which shell are you using?

_No response_

### Log output

```Shell
Version: 23.15.0

Shell: pwsh (7.4.5)

Prompt:

 username @ COMPUTERNAME   ]8;;file:D:/TEMP/E_TEMP/ohmyposh-fossil-integrate\D:\TEMP\E_TEMP\ohmyposh-fossil-integrate]8;;\   trunk +1 -1  pwsh  28/09/2024 13:21:29
❯ 

Segments:

ConsoleTitle(true)                        -   3 ms
Session(true)                             -   0 ms
Path(true)                                -   1 ms
Fossil(true)                              -  65 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms
Node(false)                               -   1 ms
Go(false)                                 -   0 ms
Python(false)                             -   7 ms
Text(true)                                -   0 ms

Run duration: 88.0016ms

Cache path: C:\Users\username\AppData\Local\oh-my-posh

Config path: D:\JoeUser\fuentes\4dos\mghsm.omp.json

Logs:

[38;2;191;207;240m[DEBUG] [38;2;156;231;201m13:21:29.547 [0mdebug.go:49[38;2;156;231;201m →[0m debug mode enabled
[DEBUG] 13:21:29.551 debug.go:49 → plain mode enabled
[DEBUG] 13:21:29.551 terminal.go:Getenv:164 → C:\Users\username\AppData\Local
[TRACE] 13:21:29.551 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 13:21:29.551 terminal.go:CachePath() - 0s
[DEBUG] 13:21:29.551 terminal.go:70 → loading cache file: C:\Users\username\AppData\Local\oh-my-posh\omp.cache
[DEBUG] 13:21:29.552 terminal.go:70 → loading cache key: upgrade_check
[TRACE] 13:21:29.552 terminal.go(C:\Users\username\AppData\Local\oh-my-posh\omp.cache) - 998.8µs
[DEBUG] 13:21:29.552 terminal.go:Getenv:164 → C:\Users\username\AppData\Local
[TRACE] 13:21:29.552 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 13:21:29.552 terminal.go:CachePath() - 0s
[DEBUG] 13:21:29.552 terminal.go:70 → loading cache file: C:\Users\username\AppData\Local\oh-my-posh\omp.cache.5ce0e11b-a664-4b11-ac8a-4029a280d6b0
[DEBUG] 13:21:29.552 terminal.go:70 → loading cache key: prompt_count_cache
[DEBUG] 13:21:29.552 terminal.go:70 → loading cache key: template_cache
[TRACE] 13:21:29.552 terminal.go(C:\Users\username\AppData\Local\oh-my-posh\omp.cache.5ce0e11b-a664-4b11-ac8a-4029a280d6b0) - 0s
[TRACE] 13:21:29.552 terminal.go:setPromptCount() - 0s
[DEBUG] 13:21:29.552 terminal.go:Getenv:164 → D:\JoeUser\fuentes\4dos\mghsm.omp.json
[TRACE] 13:21:29.552 terminal.go:Getenv(POSH_THEME) - 0s
[DEBUG] 13:21:29.552 terminal.go:ResolveConfigPath:96 → config set using POSH_THEME: D:\JoeUser\fuentes\4dos\mghsm.omp.json
[TRACE] 13:21:29.552 terminal.go:ResolveConfigPath() - 0s
[TRACE] 13:21:29.552 terminal.go:Init() - 4.9992ms
[TRACE] 13:21:29.552 terminal.go:Flags() - 0s
[TRACE] 13:21:29.553 load.go:loadConfig() - 1.0011ms
[TRACE] 13:21:29.553 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.553 debug.go:57 → terminal program: Windows Terminal
[DEBUG] 13:21:29.553 debug.go:57 → terminal shell: shell
[DEBUG] 13:21:29.553 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.553 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 13:21:29.553 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.553 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.553 terminal_windows.go:WindowsRegistryKeyValue:215 → ColorizationColor(DWORD): 0xC48F3F3F
[TRACE] 13:21:29.553 terminal.go:Shell() - 0s
[DEBUG] 13:21:29.553 terminal.go:Getenv:164 → 7.4.5
[TRACE] 13:21:29.553 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 13:21:29.553 debug.go:PrintDebug:23 → segment: Title
[DEBUG] 13:21:29.553 text.go:Render:80 → rendering template: {{ .Shell }} in {{ .Folder }}
[TRACE] 13:21:29.554 terminal.go:Flags() - 0s
[TRACE] 13:21:29.555 terminal_windows.go:Root() - 0s
[TRACE] 13:21:29.555 terminal.go:Shell() - 0s
[TRACE] 13:21:29.555 terminal.go:StatusCodes() - 0s
[TRACE] 13:21:29.555 terminal_windows.go:IsWsl() - 0s
[DEBUG] 13:21:29.556 terminal.go:TemplateCache:651 → environment: [ALLUSERSPROFILE=C:\ProgramData ... rest trimmed for privacy ...]
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:PathSeparator() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 terminal.go:Pwd:189 → D:\TEMP\E_TEMP\ohmyposh-fossil-integrate
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal_windows.go:IsWsl() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 terminal.go:User:330 → username
[TRACE] 13:21:29.556 terminal.go:User() - 0s
[DEBUG] 13:21:29.556 terminal.go:Host:347 → COMPUTERNAME
[TRACE] 13:21:29.556 terminal.go:Host() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.556 terminal.go:Getenv(SHLVL) - 0s
[TRACE] 13:21:29.556 terminal.go:TemplateCache() - 1.0009ms
[TRACE] 13:21:29.556 terminal.go:Shell() - 0s
[DEBUG] 13:21:29.556 terminal.go:Getenv:164 → 7
[TRACE] 13:21:29.556 terminal.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 13:21:29.556 terminal.go:Getenv:164 → 1
[TRACE] 13:21:29.556 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Time
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Fossil
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Path
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Root
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Git
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Session
[TRACE] 13:21:29.556 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 time.go:Enabled:29 → 15:04:05
[DEBUG] 13:21:29.556 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.556 terminal.go:Getenv(SSH_CONNECTION) - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:PathSeparator() - 0s
[DEBUG] 13:21:29.556 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.556 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.556 terminal.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 13:21:29.556 terminal.go:Shell() - 0s
[TRACE] 13:21:29.556 terminal.go:TemplateCache() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal_windows.go:Root() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 segment.go:SetEnabled:115 → segment: Shell
[DEBUG] 13:21:29.556 shell.go:Enabled:28 → mapped_shell_names: map[]
[TRACE] 13:21:29.556 terminal.go:Shell() - 0s
[TRACE] 13:21:29.556 terminal.go:Flags() - 0s
[TRACE] 13:21:29.556 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.556 path.go:setMappedLocations:563 → mapped_locations_enabled: true
[DEBUG] 13:21:29.556 path.go:setMappedLocations:564 → 
[DEBUG] 13:21:29.556 path.go:setMappedLocations:565 → 
[DEBUG] 13:21:29.556 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:PathSeparator() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.556 path.go:setMappedLocations:566 → ~
[DEBUG] 13:21:29.556 path.go:setMappedLocations:571 → mapped_locations: map[]
[TRACE] 13:21:29.556 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:PathSeparator() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:PathSeparator() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.557 path.go:makeFolderFormatMap:839 → NO DATA
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.557 path.go:setStyle:218 → full
[DEBUG] 13:21:29.557 path.go:colorizePath:740 → cycle: []
[DEBUG] 13:21:29.557 path.go:getFolderSeparator:273 → NO DATA
[DEBUG] 13:21:29.557 path.go:getFolderSeparator:275 → \
[DEBUG] 13:21:29.557 path.go:colorizePath:743 → cycle_folder_separator: false
[DEBUG] 13:21:29.557 path.go:colorizePath:744 → %s
[DEBUG] 13:21:29.557 path.go:colorizePath:746 → %s
[DEBUG] 13:21:29.557 path.go:colorizePath:747 → %s
[DEBUG] 13:21:29.557 path.go:colorizePath:748 → %s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.557 terminal.go:TemplateCache() - 0s
[TRACE] 13:21:29.557 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.557 terminal.go:StackCount() - 0s
[DEBUG] 13:21:29.557 win32_windows.go:isWriteable:303 → current user is member of S-1-1-0
[DEBUG] 13:21:29.557 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 13:21:29.557 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 13:21:29.557 terminal_windows.go:DirIsWritable() - 0s
[TRACE] 13:21:29.557 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.563 terminal.go:CommandPath:389 → D:\Exec\fossil.exe
[TRACE] 13:21:29.563 terminal.go:CommandPath(fossil.exe) - 6.9973ms
[TRACE] 13:21:29.563 terminal.go:HasCommand(fossil.exe) - 6.9973ms
[DEBUG] 13:21:29.589 terminal.go:CommandPath:389 → C:\Program Files\Git\cmd\git.exe
[TRACE] 13:21:29.589 terminal.go:CommandPath(git.exe) - 32.9988ms
[TRACE] 13:21:29.589 terminal.go:HasCommand(git.exe) - 32.9988ms
[DEBUG] 13:21:29.589 git.go:shouldDisplay:312 → fetch_bare_info: false
[TRACE] 13:21:29.589 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.589 terminal.go:ResolveSymlink:286 → D:\TEMP\E_TEMP\ohmyposh-fossil-integrate
[TRACE] 13:21:29.589 terminal.go:ResolveSymlink(D:\TEMP\E_TEMP\ohmyposh-fossil-integrate) - 0s
[ERROR] 13:21:29.590 terminal.go:HasParentFilePath:553 → CreateFile .git: El sistema no puede encontrar el archivo especificado.
[TRACE] 13:21:29.590 terminal.go:HasParentFilePath(.git) - 1.0006ms
[DEBUG] 13:21:29.622 terminal.go:RunCommand:367 ↓
    repository:   D:/TEMP/E_TEMP/ofi.fossil
    local-root:   D:/TEMP/E_TEMP/ohmyposh-fossil-integrate/
    config-db:    C:/Users/username/AppData/Local/_fossil
    checkout:     5fa05f30d764950467673b22a04b9c5b44a1e4d7 2024-09-28 16:08:02 UTC
    parent:       dac44306d774086ebfb442f5d0af7548423c56d6 2024-09-28 16:05:58 UTC
    child:        d30ad1fec4208d6ead562a6e3567ee7aa86ef8df 2024-09-28 16:10:45 UTC
    tags:         trunk
    comment:      uno dos tres (user: username)
    ADDED      cinco
    ADDED_BY_INTEGRATE cuatro
    DELETED    tres
    UPDATED_BY_INTEGRATE uno
    INTEGRATE  d30ad1fec4208d6ead562a6e3567ee7aa86ef8dfdf4a2c14db2ddc14568a0017
[TRACE] 13:21:29.622 terminal.go:RunCommand(fossil.exe status) - 59.0014ms
[TRACE] 13:21:29.622 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.622 text.go:Render:80 → rendering template:   {{.Branch}} {{.Status.String}} 
[TRACE] 13:21:29.622 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.622 text.go:Render:80 → rendering template:  {{ if .SSHSession }} {{ end }}{{ .UserName }} @ {{ .HostName }} 
[TRACE] 13:21:29.622 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.622 text.go:Render:80 → rendering template:  {{ .CurrentDate | date "02/01/2006 15:04:05" }}
[TRACE] 13:21:29.622 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.622 text.go:Render:80 → rendering template:  <b>{{ .Name }}</b> 
[TRACE] 13:21:29.622 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.622 text.go:Render:80 → rendering template:   {{ path .Path .Location }} 
[TRACE] 13:21:29.622 terminal.go:Flags() - 0s
[TRACE] 13:21:29.623 terminal.go:TemplateCache() - 0s
[TRACE] 13:21:29.623 terminal.go:TemplateCache() - 0s
[TRACE] 13:21:29.623 terminal.go:TemplateCache() - 0s
[TRACE] 13:21:29.623 terminal.go:TemplateCache() - 0s
[TRACE] 13:21:29.623 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.623 text.go:Render:80 → rendering template: powerline
[TRACE] 13:21:29.623 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.623 text.go:Render:80 → rendering template: powerline
[TRACE] 13:21:29.623 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.623 text.go:Render:80 → rendering template: {{if .Root}}green{{end}}
[TRACE] 13:21:29.623 terminal.go:Flags() - 0s
[TRACE] 13:21:29.623 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.623 text.go:Render:80 → rendering template: diamond
[TRACE] 13:21:29.623 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.624 text.go:Render:80 → rendering template: {{if .Root}}green{{end}}
[TRACE] 13:21:29.624 terminal.go:Flags() - 0s
[TRACE] 13:21:29.624 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.624 text.go:Render:80 → rendering template: diamond
[TRACE] 13:21:29.624 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.624 text.go:Render:80 → rendering template: {{ if .Status.Changed }}p:yellow{{ end }}
[TRACE] 13:21:29.624 terminal.go:Flags() - 0s
[TRACE] 13:21:29.624 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.624 text.go:Render:80 → rendering template: {{ if .Status.Changed }}cyan{{ end }}
[TRACE] 13:21:29.624 terminal.go:Flags() - 0s
[TRACE] 13:21:29.624 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.624 text.go:Render:80 → rendering template: powerline
[TRACE] 13:21:29.624 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.624 text.go:Render:80 → rendering template: {{ if .Status.Changed }}p:yellow{{ end }}
[TRACE] 13:21:29.624 terminal.go:Flags() - 0s
[TRACE] 13:21:29.625 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.625 text.go:Render:80 → rendering template: {{ if .Status.Changed }}cyan{{ end }}
[TRACE] 13:21:29.625 terminal.go:Flags() - 0s
[TRACE] 13:21:29.626 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.626 text.go:Render:80 → rendering template: powerline
[TRACE] 13:21:29.626 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.626 text.go:Render:80 → rendering template: {{ if .Status.Changed }}p:yellow{{ end }}
[TRACE] 13:21:29.626 terminal.go:Flags() - 0s
[TRACE] 13:21:29.626 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.626 text.go:Render:80 → rendering template: diamond
[TRACE] 13:21:29.626 terminal.go:Flags() - 0s
[TRACE] 13:21:29.626 terminal.go:Shell() - 0s
[TRACE] 13:21:29.626 terminal.go:Flags() - 0s
[TRACE] 13:21:29.626 terminal.go:Flags() - 0s
[TRACE] 13:21:29.626 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.626 python.go:Init:61 → context
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 terminal_windows.go:58 → C:\Users\username
[DEBUG] 13:21:29.626 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.626 terminal.go:GOOS() - 0s
[TRACE] 13:21:29.626 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.626 segment.go:SetEnabled:115 → segment: Node
[DEBUG] 13:21:29.626 segment.go:SetEnabled:115 → segment: Python
[DEBUG] 13:21:29.626 language.go:Enabled:109 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 13:21:29.626 language.go:Enabled:110 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 13:21:29.626 language.go:Enabled:117 → home_enabled: true
[DEBUG] 13:21:29.626 language.go:Enabled:110 → folders: []
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 language.go:Enabled:117 → home_enabled: false
[DEBUG] 13:21:29.626 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 language.go:Enabled:110 → folders: []
[DEBUG] 13:21:29.626 language.go:Enabled:117 → home_enabled: false
[DEBUG] 13:21:29.626 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 language.go:Enabled:129 → files
[DEBUG] 13:21:29.626 python.go:loadContext:70 → fetch_virtual_env: true
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 terminal_windows.go:58 → C:\Users\username
[DEBUG] 13:21:29.626 terminal.go:Getenv:164 → C:\Users\username\AppData\Local
[DEBUG] 13:21:29.626 language.go:Enabled:129 → files
[TRACE] 13:21:29.626 terminal.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 13:21:29.626 node.go:loadContext:56 → fetch_package_manager: false
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.626 terminal.go:HasFilesInDir(*.go) - 0s
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.626 terminal.go:HasFilesInDir(go.mod) - 0s
[DEBUG] 13:21:29.626 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.626 terminal.go:HasFilesInDir(*.js) - 0s
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.626 terminal.go:HasFilesInDir(*.ts) - 0s
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.626 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.626 terminal.go:HasFilesInDir(package.json) - 0s
[TRACE] 13:21:29.626 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.627 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.627 terminal.go:HasFilesInDir(.nvmrc) - 0s
[TRACE] 13:21:29.627 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.627 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.627 terminal.go:HasFilesInDir(pnpm-workspace.yaml) - 0s
[TRACE] 13:21:29.627 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.627 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.627 terminal.go:HasFilesInDir(.pnpmfile.cjs) - 0s
[TRACE] 13:21:29.627 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.627 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.627 terminal.go:HasFilesInDir(.vue) - 0s
[DEBUG] 13:21:29.634 terminal.go:CommandPath:389 → D:\Prog\Python312\python.exe
[TRACE] 13:21:29.634 terminal.go:CommandPath(python) - 7.9992ms
[DEBUG] 13:21:29.634 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.634 terminal.go:HasFilesInDir(pyvenv.cfg) - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.634 terminal.go:HasFilesInDir(pyvenv.cfg) - 0s
[DEBUG] 13:21:29.634 python.go:loadContext:83 → folder_name_fallback: true
[DEBUG] 13:21:29.634 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.634 terminal.go:Getenv(VIRTUAL_ENV) - 0s
[DEBUG] 13:21:29.634 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.634 terminal.go:Getenv(CONDA_ENV_PATH) - 0s
[DEBUG] 13:21:29.634 terminal.go:Getenv:164 → NO DATA
[TRACE] 13:21:29.634 terminal.go:Getenv(CONDA_DEFAULT_ENV) - 0s
[TRACE] 13:21:29.634 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.634 terminal.go:HasFilesInDir(*.py) - 0s
[TRACE] 13:21:29.634 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.634 terminal.go:HasFilesInDir(*.ipynb) - 0s
[TRACE] 13:21:29.634 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.634 terminal.go:HasFilesInDir(pyproject.toml) - 0s
[TRACE] 13:21:29.634 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFilesInDir:242 → false
[TRACE] 13:21:29.634 terminal.go:HasFilesInDir(venv.bak) - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFolder:271 → false
[TRACE] 13:21:29.634 terminal.go:HasFolder(.venv) - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFolder:271 → false
[TRACE] 13:21:29.634 terminal.go:HasFolder(venv) - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFolder:271 → false
[TRACE] 13:21:29.634 terminal.go:HasFolder(virtualenv) - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFolder:271 → false
[TRACE] 13:21:29.634 terminal.go:HasFolder(venv-win) - 0s
[DEBUG] 13:21:29.634 terminal.go:HasFolder:271 → false
[TRACE] 13:21:29.634 terminal.go:HasFolder(pyenv-win) - 0s
[TRACE] 13:21:29.634 terminal.go:Shell() - 0s
[TRACE] 13:21:29.634 terminal.go:Flags() - 0s
[TRACE] 13:21:29.634 terminal.go:Pwd() - 0s
[DEBUG] 13:21:29.634 terminal_windows.go:58 → C:\Users\username
[TRACE] 13:21:29.634 terminal.go:GOOS() - 0s
[DEBUG] 13:21:29.634 segment.go:SetEnabled:115 → segment: Text
[TRACE] 13:21:29.634 terminal.go:TemplateCache() - 0s
[DEBUG] 13:21:29.634 text.go:Render:80 → rendering template: ❯
[TRACE] 13:21:29.634 terminal.go:Flags() - 0s
[DEBUG] 13:21:29.634 text.go:Render:80 → rendering template: plain
[TRACE] 13:21:29.635 terminal.go:Flags() - 0s
[TRACE] 13:21:29.635 terminal.go:Shell() - 0s
[TRACE] 13:21:29.635 terminal.go:Flags() - 0s
[TRACE] 13:21:29.635 terminal.go:Shell() - 0s
[DEBUG] 13:21:29.635 terminal.go:Getenv:164 → C:\Users\username\AppData\Local
[TRACE] 13:21:29.635 terminal.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 13:21:29.635 terminal.go:CachePath() - 0s
[TRACE] 13:21:29.635 terminal.go:Flags() - 0s
```

--- a/src/segments/fossil.go
+++ b/src/segments/fossil.go
@@ -13,9 +13,9 @@ func (s *FossilStatus) add(code string) {
 		s.Conflicted++
 	case "DELETED":
 		s.Deleted++
-	case "ADDED":
+	case "ADDED", "ADDED_BY_INTEGRATE":
 		s.Added++
-	case "EDITED", "UPDATED", "CHANGED":
+	case "EDITED", "UPDATED", "UPDATED_BY_INTEGRATE", "UPDATED_BY_MERGE", "CHANGED":
 		s.Modified++
 	case "RENAMED":
 		s.Moved++

--- a/src/segments/fossil_test.go
+++ b/src/segments/fossil_test.go
@@ -43,12 +43,19 @@ func TestFossilStatus(t *testing.T) {
 			parent:       e8a051e6a943a26c9c33a30df8ceda069c06c174 2022-06-04 23:09:02 UTC
 			tags:         trunk
 			comment:      In the /setup_skin page, add a mention of/link to /skins, per request in the forum. (user: stephan)
-			EDITED     auto.def
-			EDITED     configure
-			ADDED      test.tst
+			CONFLICT             test.tst
+			DELETED	             test.tst
+			ADDED                test.tst
+			ADDED_BY_INTEGRATE   test.tst
+			EDITED               auto.def
+			UPDATED              test.tst
+			UPDATED_BY_INTEGRATE test.tst
+			UPDATED_BY_MERGE 	 test.tst
+			CHANGED 	         test.tst
+			RENAMED 		     test.tst
 			`,
 			ExpectedBranch: "trunk",
-			ExpectedStatus: "+1 ~2",
+			ExpectedStatus: "+2 ~5 -1 >1 !1",
 		},
 	}
 	for _, tc := range cases {

